packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -892,7 +892,7 @@ async function compileDeployInfo({
                 runs,
                 active: true,
                 model_schema: model_schema as unknown as SyncModelSchema[],
-                input: typeof flow.input === 'string' ? flow.input : flow.input ? flow.input.name : null,
+                input: typeof flow.input === 'string' ? flow.input : null,
```

Comment:
is this not a breaking change

Reply:
it was tagged as string | LegacySyncModelSchema but it's not true since a long time